src/ophyd_async/sim/pattern_generator.py

```diff
@@ -231,10 +156,7 @@ async def open_file(
         datasets = self._get_datasets()
         for d in datasets:
             self._handle_for_h5_file.create_dataset(
-                name=d.name,
-                shape=d.shape,
-                dtype=d.dtype,
-                maxshape=d.maxshape,
```

Comment:
Why has maxshape disappeared here?

Reply:
This has since been discussed in a meeting, we are going to address the larger issue of the pattern generator in a following PR. https://github.com/bluesky/ophyd-async/issues/432